spec/requests/application_html_spec.rb

```diff
@@ -0,0 +1,12 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'ApplicationHtml', type: :request do
```

Comment:
Would you mind changing this to be "the homepage" or similar and adjusting the file name?